demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss

```diff
@@ -48,6 +48,10 @@ $height-map-px: (
         height: 100vh; // .height-100vh
     }
 
+    &-44px {
```

Comment:
If you really need that "random" px value, try Tailwind with `h-[44px]`.

Reply:
I tried to use an [arbitrary](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) value here but for some reason it did not work (which I couldn't find out why)

Reply:
I removed the class here and used tailwind class `h-11` which fits exactly [5b6ce57](https://github.com/demos-europe/demosplan-core/pull/2366/commits/5b6ce57a2d6e31276391eb15750fbf31b1fa0f21)